.bashrc

```diff
@@ -80,8 +82,38 @@ if [ -x /usr/bin/dircolors ]; then
     alias egrep='egrep --color=auto'
 fi
 
+# Initiate virtualenv
 WORKON_HOME="/home/vagrant/.virtualenvs"
 source /usr/local/bin/virtualenvwrapper.sh > /dev/null 2>&1
 cd /vagrant
 
-# the workon command to be added during provisioning 
+# Ask for git configuration data from user based on input
+chmod 755 /home/vagrant/gitconfig
+export PATH=$PATH:/home/vagrant/
+echo 'export PATH=$PATH:/home/vagrant/' >> ~/.profile
+iterator1=0
+iterator2=0
+while [[ $iterator1 = 0 && $iterator2 -lt 5 && ! -f ~/.gitconfig ]]
+do
+    read -p 'Do you wish to setup Github credentials for easy access? (y/n) >>> ' github
+
+    if [ $github = 'y' ]
```

Comment:
instead of checking for `y`, check for case insensitive `y`. In fact, check for a start with type of search. Most POSIX flags are also compatible with these.

You might have noticed that writing `y` or `Y` or `yes` or `Yes` or `YES` - all work for scripts like this.